examples/fscloud/variables.tf

```diff
@@ -58,3 +58,9 @@ variable "environment_crn" {
     error_message = "Dedicated environment CRN is required to create a standard cloudant instance."
   }
 }
+
+variable "enable_cors" {
```

Comment:
Do not expose this in the example. We don't want example to become flexible solutions. I suggest to hard code it to true in the fscloud example main.tf with a code comment explaining what it does.